apps/web/src/pages/templates/shared/channels.ts

```diff
@@ -9,6 +10,7 @@ export const channels = [
     Icon: MobileGradient,
     testId: 'inAppSelector',
     channelType: ChannelTypeEnum.IN_APP,
+    action: false,
```

Comment:
Maybe this can be a type rather than a boolean flag, so it will be easier to handle new step types in the future